.github/workflows/build-and-test.yml

```diff
@@ -52,6 +56,19 @@ jobs:
                   git fetch --prune --unshallow
                   git submodule -q update --init --recursive
 
+            # - name: Setup nuget cache
+            #   uses: actions/cache@v2
+            #   id: nuget-cache
+            #   with:
+            #       path: ~/.nuget
+            #       key: ${{ runner.os }}-nuget-${{ hashFiles('**/src/ImageSharp/*.csproj') }}
+            #       restore-keys: ${{ runner.os }}-nuget-
```

Comment:
Is this something we shall try to re-enable? If not then we should probably delete it.
Or if it might be useful in the future, let's document the purpose of the commented out code (and also the reason why is it commented out).

Reply:
It's disabled temporary. I'll enable it. If you don't mind, I recommend you to use cache for packages, it's seriously speedup build time.